package.json

```diff
@@ -155,7 +155,7 @@
 		"page": "1.11.4",
 		"path-browserify": "1.0.0",
 		"percentage-regex": "3.0.0",
-		"phone": "git+https://github.com/Automattic/node-phone.git#v2.4.0-pre-1",
+		"phone": "2.3.22",
```

Comment:
There's now [v2.4.0](https://github.com/AfterShip/phone/releases/tag/2.4.0) BTW.

Reply:
Thanks, tests still pass fine, so I've bumped the version.